### PR TITLE
Add integration test for `describe-stacks`

### PR DESCRIPTION
**Pull request by Skarlso**
### Description

Closes https://github.com/weaveworks/eksctl/issues/4447

### Checklist
- [x] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [x] Manually tested
- [ ] Made sure the title of the PR is a good description that can go into the release notes
- [ ] (Core team) Added labels for change area (e.g. `area/nodegroup`) and kind (e.g. `kind/improvement`)

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

File: integration/tests/crud/creategetdelete_test.go [hunk 1]
```diff
@@ -155,6 +157,20 @@ var _ = Describe("(Integration) Create, Get, Scale & Delete", func() {
...
+				session := cmd.Run()
+				Expect(session.ExitCode()).To(BeZero())
+				var stacks []*cloudformation.Stack
+				Expect(yaml.Unmarshal(session.Out.Contents(), &stacks)).To(Succeed())
+				Expect(len(stacks) > 1).To(BeTrue())
+				for _, s := range stacks {
+					Expect(aws.StringValue(s.StackName)).To(ContainSubstring(params.ClusterName))
+				}
+			})
+		})
```
Comment (aclevername) [inline integration/tests/crud/creategetdelete_test.go:172]: does `describe-stacks` return all stacks, e.g. stack for the cluster, stack for the nodegrou, stack for an IRSA etc? If it does might be worth expanding the test a bit to check that

Comment (Skarlso) [reply integration/tests/crud/creategetdelete_test.go:172]: It returns all stacks.

Comment (aclevername) [reply integration/tests/crud/creategetdelete_test.go:172]: can we update this test to check that logic as well then?

Comment (Skarlso) [reply integration/tests/crud/creategetdelete_test.go:172]: `Expect(len(stacks) > 1).To(BeTrue())` - fails if there is only one :)

Comment (aclevername) [reply integration/tests/crud/creategetdelete_test.go:172]: > 

that isn't a very explicit test though...

Comment (Skarlso) [reply integration/tests/crud/creategetdelete_test.go:172]: But I don't wannnnaaa

tony

Comment (Skarlso) [reply integration/tests/crud/creategetdelete_test.go:172]: Fiiiiineee. I'll do it. I'll update it. Uuuuhh. Making me work on a Monday... *grumble*, *grumble*

Comment (cPu1) [reply integration/tests/crud/creategetdelete_test.go:172]: I agree that we should expand this test to ensure stacks for all resources are returned.

Comment (Skarlso) [reply integration/tests/crud/creategetdelete_test.go:172]: On it, on it. :D I'll do this right after I have the config-file thing working right. :D

**Comment by Skarlso** [comment]
```
Ran 1 of 43 Specs in 1363.688 seconds
SUCCESS! -- 1 Passed | 0 Failed | 0 Pending | 42 Skipped
PASS | FOCUSED

```

File: integration/tests/crud/creategetdelete_test.go [hunk 2]
```diff
@@ -155,6 +157,23 @@ var _ = Describe("(Integration) Create, Get, Scale & Delete", func() {
...
+		Context("and describe the stack for the cluster", func() {
+			It("should describe the cluster's stack", func() {
+				cmd := params.EksctlUtilsCmd.WithArgs("describe-stacks", "--cluster", params.ClusterName, "-o", "yaml")
+				session := cmd.Run()
+				Expect(session.ExitCode()).To(BeZero())
+				var stacks []*cloudformation.Stack
+				Expect(yaml.Unmarshal(session.Out.Contents(), &stacks)).To(Succeed())
+				Expect(len(stacks) == 2).To(BeTrue())
+				nodegroupStack := stacks[0]
+				clusterStack := stacks[1]
```
Comment (aclevername) [inline integration/tests/crud/creategetdelete_test.go:169]: is this order guaranteed?

Comment (Skarlso) [reply integration/tests/crud/creategetdelete_test.go:169]: yep

**Review by aclevername — APPROVED** [review]
lgtm- 1 nit

**Pull request merged**